cpp/src/slice2cpp/CPlusPlusUtil.cpp

```diff
@@ -607,7 +607,7 @@ Slice::opFormatTypeToString(const OperationPtr& op)
     switch (op->format())
     {
         case DefaultFormat:
-            return "::Ice::FormatType::DefaultFormat";
+            return "::std::nullopt";
```

Comment:
In the slice2xxx compilers, FormatType has still 3 values that correspond to the metadata.